src/DotNetWorker.Grpc/DotNetWorker.Grpc.csproj

```diff
@@ -21,9 +21,9 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Core" Version="1.10.0" />
+    <PackageReference Include="Azure.Core" Version="1.37.0" />
     <PackageReference Include="Google.Protobuf" Version="3.23.3" />
```

Comment:
Should we also consider upgrading Google.Protobuf to[ 3.25.2](https://www.nuget.org/packages/Google.Protobuf/3.25.2)?

Reply:
Not required as part of this pass, but I can. Will leave this unresolved and if pushing an update, will bump this up as well.